charts/super-agent/charts/super-agent-deployment/templates/secret-sa-auth.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if include "newrelic-super-agent.auth.secret.shouldTemplate" . -}}
```

Comment:
i would add a comment explaining that usually we do not render it

Reply:
I don't know if we are so sure of not rendering this so often... Why would you expect that clients will not embed the keys in the values file?